core/src/main/java/apoc/export/csv/CsvEntityLoader.java

```diff
@@ -99,7 +100,7 @@ public void loadNodes(
             final Map<String, Mapping> mapping = getMapping(fields);
 
             final CSVReader csv = new CSVReaderBuilder(reader)
-                    .withCSVParser(new CSVParserBuilder()
+                    .withCSVParser(new RFC4180ParserBuilder()
```

Comment:
Will this break any existing use cases? If so it would be nice to update documentation that we parse according to RFC 4180 now.

Reply:
I think it would affect only the backslashes, or at least I couldn't find any information suggesting that it would parse anything else differently.